functions/remove.go

```diff
@@ -0,0 +1,19 @@
+package functions
+
+// Remove items from slice when item existed
+func (ss SliceType) Remove(items ...ElementType) (result SliceType) {
+	result = SliceType{}
+	for _, v := range ss {
```

Comment:
O(n^2) can be improved upon a lot, if you convert `items` into a map first then you don't need to iterate all `items` every time. You can then call this O(n).